.circleci/post_publish.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+REPO_NAME=amplify-cli
+REPO_URL="git@github.com:aws-amplify/$REPO_NAME.git"
+GIT_USER_NAME="amplify-data-dev-git"
+GIT_USER_EMAIL="amplify-data-dev+github@amazon.com"
+FEATURE_BRANCH=guest/data/update-data-packages
+BASE_BRANCH=dev
+COMMIT_MESSAGE="chore: bump API plugin dependencies"
+PR_BODY="
+#### Description of changes
+- Bump the API and Codegen plugin dependencies to latest versions.
+
+#### Issue #, if available
+None
+
+#### Description of how you validated changes
+Build and Unit test jobs run successfully.
+
+#### Checklist
+
+- [x] PR description included
+- [x] 'yarn test' passes
+
+By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+"
+
+# Create a temporary directory
+tmp_dir=$(mktemp -d)
+curr_dir=$pwd
+cd $tmp_dir
+echo "Using Temp Directory: $tmp_dir"
+
+# Clone the CLI repo
+git clone $REPO_URL --depth 1
+cd $REPO_NAME
+
+# Checkout the feature branch to be used for version bumps
+if git show-ref --quiet refs/heads/$FEATURE_BRANCH; then
+  git branch -D $FEATURE_BRANCH # Delete if already exists locally
+fi
+git checkout -b $FEATURE_BRANCH
+
+# Pull latest changes from remote if any
+git pull
+
+# Bump the versions locally
+sudo npm install -g npm-check-updates --force
+yarn update-data-packages
```

Comment:
Won't this also update the codegen packages? Is that a concern?

Reply:
It will bump both API and Codegen plugin related dependencies. It's not a concern as today we typically bump both of them together. There are few exceptions but those are mainly for hotfixes in which case the target branch is not `dev` anymore. So those need manual intervention from on-call.